notebooks/agent_with_human_in_the_loop.ipynb

```diff
@@ -0,0 +1,602 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #38.    git_list_commits_tool = create_tool_from_function(
</pre><p>I thinks it's worth using <code>@tool</code> decorator instead of <code>create_tool_from_function</code> ﻿. For this one and for other tools as well. wdyt?</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepset-ai/haystack-cookbook/pull/194/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>